pallets/liquidity-pools-gateway/routers/src/tests.rs

```diff
@@ -315,29 +304,6 @@ mod xcm_router {
 				assert_ok!(router.do_send(test_data.sender, test_data.msg));
 			});
 		}
-
-		#[test]
-		fn transactor_info_not_set() {
```

Comment:
Thanks for removing this, it was a left-over from the time we had init logic in the router IIRC.

Reply:
My artifact. just thought it is still valid to test whether it works if global info is not set. But is implicitly anyways tested ^^